content/en/docs/started/workstation/kubeflow-on-microk8s.md

```diff
@@ -7,7 +7,9 @@ weight = 60
 
 ## Introduction
 
-This guide describes how to deploy and run Kubeflow using [MicroK8s](https://microk8s.io/) - a small enterprise Kubernetes cluster — locally on built-in hypervisors.
+This guide describes how to deploy and run Kubeflow on [MicroK8s](https://microk8s.io/) - a small enterprise Kubernetes cluster — locally on built-in hypervisors.
+
+Kubeflow comes already built-in as a MicroK8s add-on.
```

Comment:
```suggestion
MicroK8s comes with a built-in Kubeflow add-on.
```

WDYT

Reply:
Here the order of Kubeflow 1st makes sense because users are not looking for Microk8s but rather an easy way to get Kubeflow. 

"Kubeflow is already built-in as a MicroK8s add-on." ?

Reply:
Gotcha. Maybe: "Kubeflow is already built into MicroK8s as an add-on." And to make sure the paragraph(s) have cohesion, maybe we can add a little explainer so that it kind of flows:

```suggestion
Kubeflow is already built into MicroK8s as an add-on. This means once you install MicroK8s, you can start deploying Kubeflow straight away."
```

(as in `microk8s enable kubeflow`)

WDYT